stdlib/maps.mni

```diff
@@ -52,14 +60,7 @@ class Map {
     }
 
     show() {
-        let r = "{ ";
```

Comment:
Any reason why this function has been deleted?

Reply:
The += is not yet supported for strings. Only numbers. 

Once this is supported, it will go back.